scripts/vscode-build.ps1

```diff
@@ -16,8 +16,16 @@ if ($projectFileInfo) {
   $frameworkArg = if ($framework -ne "") { " -p:TargetFramework=$framework" } else { "" }
   $buildArgs = "$($buildTool.Command) -v:m -m -p:UseRoslynAnalyzers=false -p:GenerateFullPaths=true$frameworkArg $($projectFileInfo.FullName)"
 
-  Write-Host "$($buildTool.Path) $buildArgs"
-  Exec-Console $buildTool.Path $buildArgs
+  $buildToolPath = $buildTool.Path
+
+  if ($msbuildEngine -ne "vs") {
+    # InitializeBuildTool is hardcoded to dotnet.exe. If we're not building using vs (meaning msbuild), use
```

Comment:
> InitializeBuildTool is hardcoded to dotnet.exe

Why do you believe this is the case? The code for `InitializeBuildTool` will definitely use `msbuild` when the build engine is `"vs"` and it's pretty easy to demonstrate

```ps
P:\roslyn> $msbuildengine = "vs"
P:\roslyn> . .\eng\common\tools.ps1
P:\roslyn> $b = InitializeBuildTool
P:\roslyn> $b.Path
C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\msbuild.exe
P:\roslyn> 
```